cns/NetworkContainerContract.go

```diff
@@ -22,6 +22,8 @@ const (
 	UnpublishNetworkContainer                = "/network/unpublishnetworkcontainer"
 	GetInterfaceForContainer                 = "/network/getinterfaceforcontainer"
 	GetNetworkContainerByOrchestratorContext = "/network/getnetworkcontainerbyorchestratorcontext"
+	GetAllNetworkContainers                  = "/network/getallnetworkcontainers"
+	ReplenishNetworkContainer                = "/network/replenishnetworkcontainer"
```

Comment:
I get trying match the style of the existing endpoints, but we really should be making any new endpoints as good REST URLs. ([This might be a good intro doc](https://apiguide.readthedocs.io/en/latest/build_and_publish/use_RESTful_urls.html), but there is a LOT of info out there and opinions on what that means, exactly.)

As a concrete example, these GET endpoint should simply be:
```suggestion
	GetAllNetworkContainers                  = "/network/networkcontainers"
	ReplenishNetworkContainer                = "/network/networkcontainers/<something?>"
```
I'm trying to understand what `replenish` means in this context exactly, but I'm thinking that it should actually be a PUT on the same URI (`/network/networkcontainers`) with a payload containing the NCs that need to be updated - this request is coming from DNC, which has that data already, right? No need to call CNS to tell it to call DNC, from DNC.

Reply:
Addressed. Thanks for the link. It's very good to know.